README.md

```diff
@@ -96,37 +100,44 @@ Update all rule descriptions for a specific language.
 
 #### Generate new rule description
 
-To fetch static files for a rule SXXXX from RSPEC for one of the language, execute the following command:
+To fetch static files for a rule SXXXX from RSPEC for one of the languages, execute the following command:
 ```shell
 ./gradlew ruleApiUpdateRuleArm -Prule=SXXXX
 ./gradlew ruleApiUpdateRuleCloudformation -Prule=SXXXX
 ./gradlew ruleApiUpdateRuleDocker -Prule=SXXXX
 ./gradlew ruleApiUpdateRuleKubernetes -Prule=SXXXX
 ./gradlew ruleApiUpdateRuleTerraform -Prule=SXXXX
-
 ```
 
 #### Ruling integration tests
 These integration tests verify that, given a set of files, when the analyzer is run on them, all the expected issues get raised in a prepared SonarQube instance. 
 The expected findings are saved in `its/ruling/src/integrationTest/resources/expected`. To run the ruling ITS:
 - Make sure the project is built with the latest changes
-- Load/update the analyzed files: `git submodule update --init`
-- In `its/ruling` run:
-  
-  ```shell
-  ./gradlew :its:ruling:integrationTest
-  ``` 
+- Load/update the analyzed files:
 
-It is possible to keep the prepared SonarQube instance running to better inspect actual-vs-expected differences (if there are any) in the SQ UI. For this use:
+```shell
+  git submodule update --init
+```
+
+- Run:
+
+```shell
+./gradlew :its:ruling:integrationTest -p its/ruling
+``` 
+
+It is possible to keep the prepared SonarQube instance running to better inspect actual-vs-expected differences (if any) in the SQ UI. For
+this use:
   
   ```shell
-  ./gradlew :its:ruling:integrationTest -DkeepSonarqubeRunning=true
+  ./gradlew :its:ruling:integrationTest -p its/ruling -DkeepSonarqubeRunning=true
```

Comment:
The original readme asked the user to launch the command from its/ruling directory, and the command wouldn't even run since it wasn't pointing at the Gradle wrapper in root. My solution was to embed the path in the command, but I didn't know that it could run from root directly, good catch!